### DISCUSSION TITLE
React Native Support

### DISCUSSION THREAD

**Discussion by Bayramito** [katspaugh/wavesurfer.js#3514] [Ideas]
Is there any possibility to adapt this library for react native as well ?

it would be great because there is no such library out there for react native. 
The ones that are existing are for deprecated and for very old versions of react native

**Comment by katspaugh** [comment]
Thanks for the idea!

Edit:

I’ve never worked with React Native but my understanding is that it’s running without a browser environment (e.g. in JavaScriptCore), and thus wavesurfer will most likely not work there as it uses a lot of DOM and Web Audio.

**Comment by TheoOliveira** [comment]
I know this has been recent, but have you tried using?   @Bayramito

**Reply by KaleSurfer** [reply]
Appears you meant to say using X as in some library or tool? It didn't come through

**Reply by TheoOliveira** [reply]
I am sorry, i meant if it's usable in react native, since it's not something really totally incompatible. I am tight on time to try out so that's why i was asking, if was usable.
